### ISSUE TITLE
hsmm-pi see's nodes, but isn't being seen by other nodes.

### ISSUE THREAD

**Issue by flask28** [urlgrey/hsmm-pi#23]
I've got a pair of wrt54 V1 nodes running and working fine. I just did a fresh install of raspian  and hsmm-pi on my Rpi and tried to add it to the mesh - I'm getting an interesting symptom.

Neither of the two wrt54 nodes show the hsmm-pi node in their mesh status or olsrd stats pages. But the hsmm-pi node is displaying the two wrt54 nodes in both. Only by IP address, however, and no services are being presented. DNS services don't seem to be working.

The hsmm-pi is running in gateway mode. Do these symptoms suggest anything?

Thanks!

**Comment by virtualdxs** [comment]
I am having the same issue on the latest version of Raspbian, which makes sense based on the reason you gave for the bug. I see two options: 1. Tell all users to download that specific image of Raspbian or 2. Have the script downgrade the packages that need to be lower versions. 
Can this issue be reopened? I don't know how I would do that or if I even can.

**Comment by mitchbear** [comment]
SOLVED - (FOUND ISSUE FOR ME - SEE UPDATE AT THE BOTTOM OF THIS NOTE) I am also experiencing these symptoms - the HSMM-Pi can see the HSMM-Mesh node with an IP address; but the HSMM-Mesh node WRT51 v3 cannot see the HSMM-Pi.   It would be great to re-open this issue.)

Here is the old problem report:

All of my nodes are BroadbandHamnet-20-v3

The version of Raspbian recommended above is older than the Pi3 Model B - so that will not work for me.

I tried with a fresh Raspbian Jesse install, with no apt-get updates or upgrades, with the same result.

Attempted with headless install - latest Jessie Lite build, strictly following instructions, same result.

Attempted with headless on Raspberry 2, Comfast 802.11 Dongle, latest Jessie Lite build, same result.

Two failing nodes cannot see each other.

... Update: Found the discussion from William Hill on the Google+ list on this topic from March 9, 2016, and it seems that only certain Wifi Adapters work, and the ones that do not often give this symptom.  Raspberry Pi 3's need to have their internal wifi device blacklisted. Ordered an Alfa AWUS036NH and plugged in, changed the network adapter in the HSMM-Pi Network page from wlan0 to wlan1.   Rebooted - and... It works!  Just plug and play, and no need to add any commands to disable the Raspberry Pi 3 onboard wifi - it happened automatically.  I now have a mesh node (which is also a web server - just add content to /var/www/html/...).  I also tried a generic dongle from eBay and that one also works (YMMV).

The comment above regarding not to do a software upgrade seems to be a red herring - I used the latest software and had no issue.

- Mitch
KI7JQK
